Cubical/Relation/Nullary/Base.agda

```diff
@@ -0,0 +1,69 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Relation.Nullary.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Functions.Fixpoint
+
+open import Cubical.Data.Empty as ⊥
+open import Cubical.HITs.PropositionalTruncation.Base
+
+private
+  variable
+    ℓ  : Level
+    A  : Type ℓ
+
+-- Negation
+infix 3 ¬_
+
+¬_ : Type ℓ → Type ℓ
+¬ A = A → ⊥
+
+-- Decidable types (inspired by standard library)
+data Dec (P : Type ℓ) : Type ℓ where
+  yes : ( p :   P) → Dec P
+  no  : (¬p : ¬ P) → Dec P
+
+NonEmpty : Type ℓ → Type ℓ
+NonEmpty A = ¬ ¬ A
+
+Stable : Type ℓ → Type ℓ
+Stable A = NonEmpty A → A
+
+-- reexport propositional truncation for uniformity
+open Cubical.HITs.PropositionalTruncation.Base
+  using (∥_∥) public
```

Comment:
Why use this funny notation instead of the normal one for the truncation?

Reply:
Not sure I understand what you mean by "funny", can you expand?

Reply:
Haha, nevermind! It's the font on this computer that makes it look really funny... My bad